store/sqlstore/team_store.go

```diff
@@ -817,70 +817,61 @@ func (s SqlTeamStore) GetTeamsByScheme(schemeId string, offset int, limit int) s
 // in batches as a single transaction per batch to ensure consistency but to also minimise execution time to avoid
 // causing unnecessary table locks. **THIS FUNCTION SHOULD NOT BE USED FOR ANY OTHER PURPOSE.** Executing this function
 // *after* the new Schemes functionality has been used on an installation will have unintended consequences.
-func (s SqlTeamStore) MigrateTeamMembers(fromTeamId string, fromUserId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var transaction *gorp.Transaction
-		var err error
-
-		if transaction, err = s.GetMaster().Begin(); err != nil {
-			result.Err = model.NewAppError("SqlTeamStore.MigrateTeamMembers", "store.sql_team.migrate_team_members.open_transaction.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
-		defer finalizeTransaction(transaction)
+func (s SqlTeamStore) MigrateTeamMembers(fromTeamId string, fromUserId string) *model.AppError {
```

Comment:
You are returning an error here, and is expected to return certain data

Reply:
Thank you @jespino , let me work on making those changes